### PR TITLE
[CLOUDTRUST-2792] Upgrade to Keycloak 14.0.0

### PR DESCRIPTION
**Pull request by fperot74**

### CODE REVIEW

File: keycloak-wsfed/src/main/resources/theme/wsfed/admin/resources/partials/client-detail.html [hunk 1]
```diff
@@ -536,6 +590,91 @@
                 <kc-tooltip>{{:: 'access-token-lifespan.tooltip' | translate}}</kc-tooltip>
             </div>
 
+            <div class="form-group" data-ng-show="protocol == 'saml'">
+                <label class="col-md-2 control-label" for="samlAssertionLifespan">{{:: 'saml-assertion-lifespan' | translate}}</label>
```
Comment (fperot74) [inline keycloak-wsfed/src/main/resources/theme/wsfed/admin/resources/partials/client-detail.html:656]: Should we set it for WSFED too?

**Comment by topelrapha** [comment]
Is there anyway to support this PR?
We have to implement a solution for SharePoint and Exchange with Keycloak 14.0.

**Comment by fperot74** [comment]
> Is there anyway to support this PR?
> We have to implement a solution for SharePoint and Exchange with Keycloak 14.0.

Hi @topelrapha ,

This component is now out of scope for our project.
This PR won't be merged as we don't have time to work for this module but I keep it open in case someone would like to go on working on it.

**Comment by amarkevich** [comment]
are any issues using this changes?

**Comment by fperot74** [comment]
@amarkevich This PR is just the state of the project when we abandonned it. It can't be built and have not been tested.

**Pull request closed without merging**